routes/matches/matches.js

```diff
@@ -525,24 +547,41 @@ router.get("/:match_id", async (req, res, next) => {
       req.user !== undefined &&
       (matchRow[0].user_id == req.user.id || Utils.superAdminCheck(req.user))
     ) {
-      sql = "SELECT * FROM `match` where id=?";
+//      sql = "SELECT * FROM `match` where id=?";
+	sql =
```

Comment:
Same as above, what will happen if all values in the team is deleted? Will this just return null? What if it's in the match table (per how pugs operate) compared to null data?